### DISCUSSION TITLE
cpu temp not displaying

### DISCUSSION THREAD

**Discussion by ehula** [fastfetch-cli/fastfetch#1307] [Q&A]
`fastfetch -h cpu-format` shows the following value:
```
{8}: Temperature (formatted) - temperature
```
however, when I try using it, the temperature is not displayed.

If I try `fastfetch -c all`, the cpu temperature is shown. How can I get the cpu temperature to display using a custom format?

**Comment by CarterLi** [comment]
You need --cpu-temp

**Comment by ehula** [comment]
I'm not sure what you mean. I have configured fastfetch using home-manager module in NixOS. It looks like a json config file:
```
              {
                  type = "cpu";
                  format = "{1} {8}";
              }
```
Do I need to make a change to the above?

**Comment by CarterLi** [comment]
`temp = true`, maybe

**Comment by ehula** [comment]
Thanks for the reply, but I am still at a loss. I don't know what you mean by the above. Is there somewhere I could read up on this to make myself more informed?

**Comment by CarterLi** [comment]
I mean
```
              {
                  type = "cpu";
                  format = "{1} {8}";
                  temp = true
              }
```

**Comment by ehula** [comment]
Yes. That worked! Thank you very much for the quick help.
